relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -275,9 +294,27 @@ impl TransactionExtractor<'_> {
                     None => continue,
                 };
 
+                // We treat a measurement as "performance score" if its name is the name of another
+                // measurement prefixed by `score.`.
+                let is_performance_score = name == "score.total"
+                    || name
+                        .strip_prefix("score.weight.")
+                        .or_else(|| name.strip_prefix("score."))
+                        .map_or(false, |suffix| measurement_names.contains(suffix));
```

Comment:
This is clumsy: It would be better to mark measurements as "performance score" somehow, but they might originate from a downstream relay so any kind of flag would have to be serialized into the event JSON.